opendevin/runtime/client/runtime.py

```diff
@@ -145,7 +144,7 @@ async def _init_container(
                 )
                 volumes = None
 
-            logger.info(f'run_as_devin: `{self.config.run_as_devin}`')
+            logger.info(f'_init_container run_as_devin: {self.config.run_as_devin}')
```

Comment:
```suggestion
            logger.debug(f'EventStreamRuntime config:\n{self.config}')
```

Reply:
Forgot to remove that one, not needed anymore.